package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Minimal Apostrophe Boilerplate",
   "main": "app.js",
   "scripts": {
+    "start": "node app.js",
     "build": "node app apostrophe:generation",
     "docker-backup-content": "docker-compose exec apostrophe ./scripts/docker-backup-content",
     "docker-restore-content": "docker-compose exec apostrophe ./scripts/docker-restore-content"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/apostrophecms/apostrophe-boilerplate"
+    "url": ""
   },
   "author": "Apostrophe Technologies, Inc.",
   "license": "MIT",
   "dependencies": {
-    "apostrophe": "github:apostrophecms/apostrophe#ensure-group-race-condition"
+    "apostrophe": "^2.116.0"
```

Comment:
How about changing this to just `github:apostrophecms/apostrophe` now that the fix has merged, so experimentation can continue?